tests/test_catalog.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import pytest
+from tests._utils import get_time_str_with_random_suffix
+
+import awswrangler as wr
+
+
+@pytest.fixture(scope="function")
+def random_glue_database():
+    database_name = get_time_str_with_random_suffix()
+    yield database_name
+    wr.catalog.delete_database(database_name)
```

Comment:
Should we move this to `conftest.py` to benefit other tests with similar requirements in the future? Unless there is a reason that it must be within `test_catalog.py`?